cfme/tests/test_rest.py

```diff
@@ -22,7 +22,7 @@
 
 
 @pytest.fixture(scope="module")
-def provider():
+def a_provider():
```

Comment:
We should maybe make a fixture for setup_an_infra and setup_a_cloud provider.....what do you think?